static/js/pages/superuser.js

```diff
@@ -67,8 +67,8 @@
                 return result.toString() + " " + byte_unit;
             }
         }
-        return null
-      };
+      return (bytes / $scope.disk_size_units["MB"]).toFixed(2).toString() + " MB";
```

Comment:
Nice!
nit: If the size is too small, we might have to convert to "KB" as well. Can handle this in the UI enhancements PR.

Reply:
Added